### PR TITLE
Fixes chameleon laser guns blowing up weldertanks/not having nodamage

### PR DESCRIPTION
**Pull request by silicons**
bikehorn

### CODE REVIEW

File: code/modules/clothing/chameleon.dm [hunk 1]
```diff
@@ -476,7 +476,7 @@
 	if(!istype(P))
 		CRASH("[P] is not /obj/item/projectile!")
 		return FALSE
-	chameleon_projectile_vars = list("name" = "practice laser", "icon" = 'icons/obj/projectiles.dmi', "icon_state" = "laser")
+	chameleon_projectile_vars = list("name" = "practice laser", "icon" = 'icons/obj/projectiles.dmi', "icon_state" = "laser", "nodamage" = 1)
```
Comment (lzimann) [inline code/modules/clothing/chameleon.dm]: use TRUE

**Comment by PKPenguin321** [comment]
thanks for ANNOUNCING THAT EXPLOIT TO THE ENTIRE SERVER
foolish idlemin

**Comment by optimumtact** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Comment by kevinz000** [comment]
@optimumtact ;-;

**Pull request merged**